src/components/ContextMenu.scss

```diff
@@ -32,6 +32,13 @@
 
     display: grid;
     grid-template-columns: 1fr 0.2fr;
+
+    &.context-menu-delete {
```

Comment:
let's introduce some sort of `dangerous` class and not delete specific, so it will be future proof to add more red things :) (I doubt that it will be the case, but you never know)